packages/lib-react-components/src/ZooHeader/ZooHeader.stories.js

```diff
@@ -21,6 +22,60 @@ export default {
   }
 }
 
+export function ThemeToggleSignedIn({ signIn, signOut }) {
+  const [themeMode, setThemeMode] = useState('light')
+  function onThemeChange() {
+    let newTheme
+    if (themeMode === 'light') {
+      newTheme = 'dark'
+    } else {
+      newTheme = 'light'
+    }
+
+    setThemeMode(newTheme)
+  }
+
+  return (
+    <ZooHeader
+      onThemeChange={onThemeChange}
+      showThemeToggle
+      signIn={signIn}
+      signOut={signOut}
+      themeMode={themeMode}
+      user={{
+        display_name: 'zootester1',
+        login: 'zootester1'
+      }}
+    />
+  )
+}
+
+export function ThemeToggleSignedOut({ signIn, signOut }) {
+  const [themeMode, setThemeMode] = useState('light')
+  function onThemeChange() {
```

Comment:
This function can probably be extracted as a helper... I've seen this logic 3 times so far so probably best to extract it / import it instead of copy/paste.